build.gradle

```diff
@@ -329,40 +327,70 @@ project('spring-rabbit-test') {
 
 }
 
+configurations {
+	docs
+}
+
+def docResourcesVersion = '0.1.0.RELEASE'
```

Comment:
This should go to `ext {` block instead.

Reply:
Fixed. And now I know what the ext block is for. Thanks. :)